test/e2e/cluster_samples_operator_test.go

```diff
@@ -398,6 +414,15 @@ func verifyTemplatesPresent(t *testing.T, content map[string]bool, timeToCompare
 				t.Log(errstr)
 				return false, fmt.Errorf(errstr)
 			}
+			tv, ok := template.Annotations[samplesapi.SamplesVersionAnnotation]
```

Comment:
@bparees here is where the version check starts for templates

Reply:
ah hah, thanks